packages/eas-cli/src/credentials/ios/api/graphql/mutations/AppleAppIdentifierMutation.ts

```diff
@@ -17,14 +17,20 @@ const AppleAppIdentifierMutation = {
       graphqlClient
         .mutation<{ appleAppIdentifier: { createAppleAppIdentifier: AppleAppIdentifier } }>(
           gql`
-            mutation AppleAppIdentifierMutation($appleAppIdentifierInput: AppleAppIdentifierInput!, $accountId: ID!) {
+            mutation AppleAppIdentifierMutation(
+              $appleAppIdentifierInput: AppleAppIdentifierInput!
+              $accountId: ID!
+            ) {
               appleAppIdentifier {
-                createAppleAppIdentifier(appleAppIdentifierInput: $appleAppIdentifierInput, accountId: $accountId) {
-                  ...${AppleAppIdentifierFragment.name}
+                createAppleAppIdentifier(
+                  appleAppIdentifierInput: $appleAppIdentifierInput
+                  accountId: $accountId
+                ) {
+                  ...AppleAppIdentifierFragment
                 }
               }
             }
-            ${print(AppleAppIdentifierFragment.definition)}
+            ${print(AppleAppIdentifierFragmentDoc)}
```

Comment:
I added the suffix for a couple reasons:
1. It's the suffix I've seen the `typescript-urql` plugin add to fragment [variables](https://graphql-code-generator.com/docs/plugins/typescript-urql) (fragmentVariableSuffix)
2. Most queries that use a fragment will do something like this:
```
async fooQuery(): Promise<FooFragment> {
    urqlClient.query(
       gql`<query ast>
           ... ${FooFragmentDoc}
       `
    )
}
```
The important part is that FooFragmentDoc is named something different from the autogenerated type, FooFragment. Otherwise you'll have to do `import as` to resolve the duplicate name. 

I dont really know if the `Doc` suffix is a common convention (other than the one time i saw a plugin do it somewhere), but if `*Node` gives you a better idea about what the object is, let's just go with that.